Improved dice capabilities

VelkanGk opened the pull request:
-  `d.js` renamed to `r.js` in order to avoid crossing commands.
- removed `diceCalculationUtilities.js`
- added rule in `index.js` to allow shorthand dice call `!#d#` or `!r #d#`
- added fancy messages utility `prettyMessages.js`

Commit:
Improvements to roll command and a new command that allows to change the prefix

Commit:
settings removed until roles management

LilithTheSuccubus commented:
Sorry for the delay, I was sick last week and couldn't take a look into it. Thanks a lot for contributing!

VelkanGk commented:
@LilithTheSuccubus Don't worry, hope you are ok now!
In the mean time, I have gone through some of your issues and fixed them. If you like, you can check my approach in a second version of this bot [vgk-discord-bot](https://github.com/VelkanGk/vgk-discord-bot). I didn't create a pull request since I have changed several pieces of code and added others.
